cavern/src/main/java/org/opencadc/cavern/nodes/FileSystemNodePersistence.java

```diff
@@ -452,6 +465,14 @@ public Node put(Node node) throws NodeNotSupportedException, TransientException
                     throw new UnsupportedOperationException("link to external resource", ex);
                 }
             }
+        } else if ((node instanceof ContainerNode) && isAllocation((ContainerNode) node)
+                && node.getProperty(VOS.PROPERTY_URI_QUOTA) == null) {
+            final NodeProperty defaultNodeProperty = this.defaultProperties.get(this.defaultProperties.indexOf(new NodeProperty(VOS.PROPERTY_URI_QUOTA)));
+            if (defaultNodeProperty == null) {
+                log.warn("No default quota (" + VOS.PROPERTY_URI_QUOTA + ") configured.");
+            } else {
+                node.getProperties().add(new NodeProperty(VOS.PROPERTY_URI_QUOTA, defaultNodeProperty.getValue()));
```

Comment:
I'd prefer to keep these sanity checks independent, so 
```
if (node instanceof LinkNode) {
    // check target acceptability
}

if (...) {
    // add default quota
}
```

`log.warn`  is probably wrong: no default is a valid config, certainly with the (default) NoOpQuotaPlugin and some people might want that with the Ceph plugin. I might have a `log.debug` but it should mention the config property, not the node property identifier (uri).